### PR TITLE
Changelog for v1.6.11

### PR DESCRIPTION
**Pull request by dennisreimann**
Updating the changelog for a v1.6.11 release, which would include everything on master, so that we catch up with the current state. v1.6.10 was a release that contained cherry-picked commits.

### CODE REVIEW

**Comment by NicolasDorier** [comment]
I really want this one as well https://github.com/btcpayserver/btcpayserver/pull/4074

**Comment by dennisreimann** [comment]
How shall we proceed here? @Kukks rightfully mentioned, that the changelog is pretty big for an in between release. The original idea was to have something that catches up with master again, but I agree, there‘s a lot of potential stuff in here.

**Comment by Kukks** [comment]
My main concern is the app plugin change, which is still uncovering bugs as we go progress

**Comment by NicolasDorier** [comment]
@Kukks @dennisreimann I will cherry-pick only the bug fixes then

**Comment by NicolasDorier** [comment]
I isolated the bug fix https://github.com/btcpayserver/btcpayserver/pull/4150

File: Changelog.md [hunk 1]
```diff
@@ -26,7 +61,7 @@
 
 * Add no rate found error message in Invoices (#4039) @HamroRamro
 * Greenfield API docs improvements (#4041 #4035) @dennisreimann
-* Allow binding ip and port for non ssl scenarios too (#4031) @Kukks 
+* Allow binding ip and port for non ssl scenarios too (#4031) @Kukks
```
Comment (NicolasDorier) [inline Changelog.md:64]:
```suggestion
* Allow binding ip and port for non ssl scenarios too (#4031) @Kukks
* Fix pagination of wallet's transactions (#4150) @dennisreimann @NicolasDorier 
```

File: Changelog.md [hunk 2]
```diff
@@ -1,4 +1,39 @@
...
+
+### Improvement
+
+* Make POS and Crowdfund system plugins (#3987) @dennisreimann
+* Enhance export function for invoices (#4060) @HamroRamro
+* Create dynamic manifest for pos apps (#4064) @Kukks
+* Update invoice amount description in Swagger template @bolatovumar
+* Improve payout processors description (#4109) @woutersamaey
+* Set explicit cursor style property on pay button with custom text (#4107) @bolatovumar
+* Use mempool as default block explorer (#4100) @junderw
```
Comment (NicolasDorier) [inline Changelog.md:18]:
```suggestion
* Use mempool space as default block explorer (#4100) @junderw
```

**Comment by pavlenex** [comment]
We should proabably include https://github.com/btcpayserver/btcpayserver/pull/4161 and https://github.com/btcpayserver/btcpayserver/pull/4160

**Pull request merged**